examples/bri-3/src/bri/transactions/api/dtos/request/updateTransaction.dto.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { plainToInstance } from 'class-transformer';
+import { validate } from 'class-validator';
+import { UpdateTransactionDto } from './updateTransaction.dto';
+
+describe('UpdateTransactionDto', () => {
+  it('should return error in case payload not provided.', async () => {
+    // Arrange
+    const dto = { signature: '2323' };
+    const updateTransactionDto = plainToInstance(UpdateTransactionDto, dto);
+
+    // Act
+    const errors = await validate(updateTransactionDto);
+
+    // Assert
+    expect(errors.length).toBe(1);
+    expect(errors[0].property).toEqual('payload');
+    expect(errors[0].constraints.isNotEmpty).toContain(
+      'payload should not be empty',
```

Comment:
can we read this from constants file to avoid code change if this string is modified in actual code

Reply:
done